src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const price = 40;
+  const a = 20;
+  const b = 50;
```

Comment:
- a and b are bad names for variables, try to provide more descriptive name
- it's better to use UPPER_CASE for static constants